makewebpackconfig.js

```diff
@@ -42,6 +42,7 @@ module.exports = function(options) {
 
   // If app is in development
   } else {
+    devtool = 'cheap-eval-source-map';
```

Comment:
@mxstbr heres the line I added to webpack to create the sourcemap.. this is going to enhance debugging because debugging build isn't the best approach for development environment.